tutorial6_hierarchical/demo/main.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * CONSTANTS AND GLOBALS
+ * */
+const width = window.innerWidth * 0.9,
+  height = window.innerHeight * 0.7,
+  margin = { top: 20, bottom: 50, left: 60, right: 40 },
+  tooltipSpecs = { width: 100, height: 50 };
```

Comment:
will this work without setting the height? so that we can take advantage of div automatic re-sizing?

Reply:
I included this in case we ever want the tooltip to be in the top right quadrant, but didn't implement. Our tooltip is always in the bottom right, which is fine for now I suppose